website/docs/build-apps/how-to-guides/display-search-and-filter-list-data.md

```diff
@@ -0,0 +1,67 @@
+---
+description: This page shows you how to display and lookup data in a List widget. 
+---
+
+# Display and Lookup Data in List
+
+<div style={{ position: "relative", paddingBottom: "calc(50.520833333333336% + 41px)", height: "0", width: "100%" }}>
+  <iframe src="https://demo.arcade.software/h3qV8jRXuy6LcNNIjeEo?embed" frameborder="0" loading="lazy" webkitallowfullscreen mozallowfullscreen allowfullscreen style={{ position: "absolute", top: "0", left: "0", width: "100%", height: "100%", colorScheme: "light" }} title="Appsmith | Connect Data">
+  </iframe>
+</div>
+
+This page shows you how to display and lookup data in a List widget.
+## Prerequisites
+- A datasource containing the data to display and filter. See [Connect datasource](/getting-started/tutorials/the-basics/connect-query-display-data#connect-datasource) for a tutorial. For the list of datasources supported by Appsmith, see [Datasources](/connect-data/reference).
+
+## Display data
+To display data in a List widget, follow these steps:
+1. In the widget's property pane, set the **Items** property under **Data**.
```

Comment:
post this step, don't users have to individually configure the list item widgets using `currentItem`. that is missing both in arcade and here